From f096382e06d40cf2cf848ff9c62240fbb46a9976 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 25 Jun 2020 23:53:54 -0400
Subject: [PATCH] mtkfb: Default to RGB order

The fine folks at Mediatek even left us an ifdef to use!
---
 drivers/misc/mediatek/video/mt6785/videox/mtkfb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/mediatek/video/common/mtkfb.c b/drivers/misc/mediatek/video/common/mtkfb.c
index 39f934b1b0d7..6454d6cc5fe9 100644
--- a/drivers/misc/mediatek/video/common/mtkfb.c
+++ b/drivers/misc/mediatek/video/common/mtkfb.c
@@ -1682,7 +1682,7 @@ static int mtkfb_fbinfo_init(struct fb_info *info)
 
 	var.transp.offset = 24;
 	var.red.length = 8;
-#if 0
+#if 1
 	var.red.offset = 16;
 	var.red.length = 8;
 	var.green.offset = 8;
